lib/wcc/contentful/simple_client.rb

```diff
@@ -35,6 +36,39 @@ def get(path, query = {})
         get_http(url, query))
     end
 
+    ADAPTERS = {
+      http: ['http', '> 1.0', '< 3.0'],
+      typhoeus: ['typhoeus', '~> 1.0']
+    }.freeze
+
+    def self.load_adapter(adapter)
+      case adapter
+      when nil
+        ADAPTERS.each do |a, spec|
+          begin
+            gem(*spec)
```

Comment:
I dig the auto discovery... whatever they have installed
![](https://i.pinimg.com/originals/03/41/eb/0341eb5d8fc0ba89b5f053c7db291053.jpg)